modules/flyte/identities.tf

```diff
@@ -0,0 +1,31 @@
+resource "azurerm_user_assigned_identity" "flyte_controlplane" {
+  name                = "flyte_controlplane"
+  location            = var.azurerm_resource_group_location
+  resource_group_name = var.azurerm_resource_group_name
+  tags                = var.tags
+}
+
+resource "azurerm_federated_identity_credential" "flyte_controlplane" {
+  name                = "flyte_controlplane"
+  resource_group_name = var.azurerm_resource_group_name
+  audience            = ["api://AzureADTokenExchange"]
+  issuer              = var.azurerm_kubernetes_cluster_oidc_issuer_url
+  parent_id           = azurerm_user_assigned_identity.flyte_controlplane.id
+  subject             = "system:serviceaccount:${var.namespaces.platform}:flyte_controlplane"
```

Comment:
I'm not sure how this mapping works in azure, but if it is the same as in aws then we need this to point to the service accounts for specific flyte resources. In this case that would be things like `flyteadmin` and `flytepropeller` .

Reply:
Nice catch, thanks!